native/dlabad.go

```diff
@@ -0,0 +1,11 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+func (impl Implementation) Dlabad(small, large float64) {
```

Comment:
That was the netlib comment for the original LAPACK routine. Since parameters are passed by reference, "return" really means "overwrite". This doesn't apply since Go guarantees IEEE-754. We could also delete it entirely.